src/like-service/Dockerfile

```
@@ -2,15 +2,28 @@ FROM php:8.0-cli
 
 COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
 
-COPY . /var/www
-
 WORKDIR /var/www
 
+RUN apt update
+
+RUN apt -y install git
+
 ENV COMPOSER_ALLOW_SUPERUSER=1
 
+COPY . /var/www
+
+#This is faster because reasons https://github.com/composer/composer/issues/8205#issuecomment-507256979
```

[ISSUE] Please change this to a proper comment
```suggestion
#This is faster https://github.com/composer/composer/issues/8205#issuecomment-507256979
```

---

Changed